core/templates/dev/head/pages/exploration_player/feedback_popup_directive.html

```diff
@@ -6,6 +6,12 @@
     resize: none;
     width: 200px;
   }
+  .checkbox {
```

Comment:
rename this class to something unique so that it should not affect other places, start the class name with `oppia-`. (Suggestion: `oppia-stay-anonymous-checkbox`)

Reply:
@DubeySandeep If I change the name of the class. The UI completely changes into something like this but I get the expected changes when the class name is just .checkbox
![screenshot from 2019-02-16 09-02-13](https://user-images.githubusercontent.com/43813343/52894082-021b2900-31ca-11e9-8835-e84c6c615183.png)

Reply:
@jeeb0xical I have added a comment below PTAL, maybe that would help.

Reply:
> @DubeySandeep If I change the name of the class. The UI completely changes into something like this but I get the expected changes when the class name is just .checkbox
> ![screenshot from 2019-02-16 09-02-13](https://user-images.githubusercontent.com/43813343/52894082-021b2900-31ca-11e9-8835-e84c6c615183.png)

After making the required changes, also the output still remains to be the same @DubeySandeep . Can someone help me with this. Thanks !

Reply:
@jeeb0xical, Sorry I don't understand what you are asking. Also, I don't see this comment resolved till now!

Reply:
@DubeySandeep When i change the css class name .checkbox to class name as .oppia-stay-anonymous-checkbox the output, that is the stay anonymous in the web page turns to bold letters and of a different size. So, when the class name is just kept to be .checkbox and the styling nits is done in a separate css class, I can get the desired output